snet-ui/SubscribeToNotification/index.tsx

```diff
@@ -124,10 +124,10 @@ function SubscribeToNotification(
           </Grid>
         </Grid>
       </Box>
-      <Box sx={{ display: "flex", justifyContent: "center", mt: 3 }}>
+      <Box sx={{ display: "flex", justifyContent: "center", mt: 3,  ml: 3.3}}>
         {alertMessage.message ? (
-          <Alert severity={alertMessage.severity} sx={{ width: "47.3%" }}>
-            {alertMessage.message}
+          <Alert severity="error" sx={{ width: "49.8%", height: "45px"}}>
```

Comment:
Will this width work on all screens [Mobile / Tablet / Web] ?

Reply:
Yes it works according to screen